jobs/build/promote/Jenkinsfile

```diff
@@ -333,6 +333,19 @@ node {
                     errata_url = retval.errataUrl
                 }
             }
+            stage("add cve flaw bugs") {
+                if (advisory == -1) {
+                    return
+                }
+                if (major == 4 && !is_4stable_release) {
+                    return
+                }
+                release.getAdvisoryIds().each {
```

Comment:
Trying to understand this step, does getAdvisoriesIds() return ["image","rpm","extras"]? Kind of a nitpick but if we're reading group.yml in a previous step, might as well pass in advisories explicitly rather than ask elliott to read again?

Reply:
It gets the advisory IDs. See https://github.com/openshift/aos-cd-jobs/pull/2573/files#diff-d2bd219940ec77d5275acd297f229e62357751e91311290812987988a687c9d2R237.